packages/event-handler/src/rest/middleware/compress.ts

```diff
@@ -0,0 +1,58 @@
+import type { Middleware } from '@aws-lambda-powertools/event-handler/types';
+import type { CompressionOptions } from 'src/types/rest.js';
```

Comment:
This should also be a relative import starting with `../` - this is because the module resolution is always relative to the file that is importing, not the package root - despite what VSCode auto-suggests.